udfs/remote_udfs/nlp/README.md

```diff
@@ -0,0 +1,119 @@
+# Calling Google's Natural Language API Sentiment Analysis Example 
+
+Google's Natural Language API Sentiment Analysis inspects the given text and identifies the prevailing emotional opinion within the text, especially to determine a writer's attitude as positive, negative, or neutral.  
+
+To run this example, you will need the following APIs enabled:
+* [BigQuery API](https://cloud.google.com/bigquery/docs)
+* [Cloud Functions API](https://cloud.google.com/functions/docs)
+* [BigQuery Connections API](https://cloud.google.com/bigquery/docs/working-with-connections#enable_the_connection_service)
+* [Natural Language API](https://cloud.google.com/natural-language/docs/setup#api)
+
+## Running the example
+
+### Setting the environment variables 
+
+Replace the various environment variables below with your desired values.
+```
+PROJECT=your_project_id
+LOCATION=your_bigquery_dataset_location
+DATASET=your_bigquery_dataaset_id
+```
+
+### Navigate to the remote_udfs directory
+Change to the remote_udfs directory after cloning the repo. 
+```
+cd bigquery-utils/udfs/remote_udfs/nlp
+```
+
+### Creating your BigQuery connection 
+
+To create your BigQuery connection, you need to specify the following:
+* project_id - the project id that you wish to create the connection in
+* location - the location for the external connection in BigQuery
+
+```
+bq mk --connection --display_name=\'remote_connection\' --connection_type=CLOUD_RESOURCE \
+      --project_id=$PROJECT --location=$LOCATION remote_connection
+```
+
+### Deploying your Cloud Function
+
+[More information about deploying your cloud function can be found here.](https://cloud.google.com/functions/docs/deploy)
+
+Snippet provided below for brevity.  
+It’s recommended that you keep the default authentication instead of allowing unauthenticated invocation of your Cloud Function or Cloud Run service.  
+We use gen1 Cloud Functions here for the simple demo purposes; however, gen2 Cloud Functions are recommended.  
+
+* project - the project the cloud function is deployed to 
+* runtime - this was defaulted to python39 but can be changed as required 
+```
+gcloud functions deploy analyze-sentiment \
```

Comment:
Need to specify the location in the cmd right?

Reply:
Whoops good catch, I have a default set I think

added